ocs_ci/templates/workloads/pgsql/Benchmark.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: pgbench-benchmark
+  namespace: my-ripsaw
+spec:
+  workload:
+    name: "pgbench"
+    args:
+      timeout: 5
+      clients:
+        - 1
+        - 2
+      threads: 1
+      transactions: 10
+      cmd_flags: ''
+      init_cmd_flags: ''
+      scaling_factor: 1
+      samples: 2
+      databases:
```

Comment:
With the above values, IO runs for ~20 seconds. 

IO's ran for ~600 seconds with the below config. pls, consider updating with these values. 

    args:
      timeout: 5
      clients:
        - 10
      threads: 2
      transactions: 1000
      cmd_flags: ''
      init_cmd_flags: ''
      scaling_factor: 2
      samples: 10

Reply:
These configuration means for a basic pgsql which ensures the work flow is good.  We can change these parameters for a longer I/O run in the following scale test cases.

Reply:
Agreed to change to 60 seconds.

Reply:
Fixed.  Changed pgbench configs to run for ~60 secs.
''' args:
      timeout: 5
      clients:
        - 20
      threads: 2
      transactions: 100
      cmd_flags: ''
      init_cmd_flags: ''
      scaling_factor: 2
      samples: 5
'''

Reply:
Nice!